README.md

```diff
@@ -292,7 +292,7 @@ mContext.acquireTokenSilent(String resource, String clientId, String userId, fin
 	* If target version is lower than 23, calling app has to have the following permissions declared in manifest(http://developer.android.com/reference/android/accounts/AccountManager.html):
 		* GET_ACCOUNTS
 		* USE_CREDENTIALS
-		* MANAGE_ACOUNTS
+		* MANAGE_ACCOUNTS
```

Comment:
As much as i like this, no unrelated changes in a PR, you can make this change when you fix #898